### PR TITLE
Added SuperSCS interface + changes to SizeMetrics

### PR DESCRIPTION
**Pull request by nishi951**

### CODE REVIEW

File: cvxpy/problems/solvers/superscs_intf.py [hunk 1]
```diff
@@ -0,0 +1,183 @@
...
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+
+import cvxpy.settings as s
+from cvxpy.problems.solvers.ecos_intf import ECOS
+import numpy as np
+
+
+class SUPERSCS(ECOS):
```
Comment (SteveDiamond) [inline cvxpy/problems/solvers/superscs_intf.py]: Can't you use the SCS interface as a base class and have less replication?

**Comment by nishi951** [comment]
Oh yeah that's clearly better.

Unrelated, but I also changed the SizeMetrics object to be a collections.namedtuple instead of a full-fledged object - this should make it more compact in memory and also a little easier to maintain, while essentially preserving all the original syntax.

File: cvxpy/problems/solvers/superscs_intf.py (hunk 1 above)
Comment (nishi951) [reply cvxpy/problems/solvers/superscs_intf.py]: Yes definitely, didn't think of that...

**Comment by SteveDiamond** [comment]
Looks good. Merged now.

**Pull request merged**